libraries/botbuilder-dialogs-declarative/src/customDeserializer.ts

```diff
@@ -6,6 +6,6 @@
  * Licensed under the MIT License.
  */
 
-export interface Converter {
-    convert(value: any): any;
+export interface CustomDeserializer {
```

Comment:
It would be great to parameterize this:

```typescript
export interface CustomerDeserializer<T, C> {
  load(config: C, type: new (...args: unknown[]): T): T;
}
```